QUARKUS-1398.md

```diff
@@ -0,0 +1,34 @@
+# QUARKUS-1398 - Test and verify GraphQL support
+
+JIRA link: https://issues.redhat.com/browse/QUARKUS-1398
+
+Coverage should ensure full GraphQL support from client and server side.
+For native mode similar test coverage as for JVM mode needs to be ensured.
+
+## Scope of the testing
+Quarkus supported GraphQL for a long time as a tech preview and our tests covered only basic functionality, but since we move towards full support, following actions are needed to ensure test coverage:
```

Comment:
One again, what is 'basic functionality'? I don't particularly like nebulous language in test plans. Do we have a test plan for previous tech support of GraphQL? Maybe this could be referenced here.

Reply:
I changed the description